src/main/java/org/runejs/Configuration.java

```diff
@@ -52,6 +57,47 @@ public static void read() {
 
         } catch (Exception e) {
             System.out.println("Unable to load client config - using defaults.");
+            Map<String, Object> net = new HashMap<String, Object>();
+
+            net.put("address", SERVER_ADDRESS);
+            net.put("game_port", GAME_PORT);
+
+            Map<String, Object> cache = new HashMap<String, Object>();
+            cache.put("cacheDir", GAME_PORT);
+
+            Map<String, Object> rsa = new HashMap<String, Object>();
+            rsa.put("rsaPub", RSA_PUBLIC_KEY);
+            rsa.put("rsaModulus", RSA_MODULUS);
+
+            Map<String, Object> login = new HashMap<String, Object>();
+            login.put("useStaticCredentials", USE_STATIC_DETAILS);
+            login.put("username", USERNAME);
+            login.put("password", PASSWORD);
```

Comment:
Is this the user's typed password? Not sure we should be saving this if so

Reply:
This is only the default password, reading from Configuration.PASSWORD on startup, so it will not write the typed password